### ISSUE TITLE
Android groups applications on UID

### ISSUE THREAD

**Issue by Halilb1978** [OxfordHCC/tracker-control-android#229]
System applications  have the same UID, even they have a different package and application name. Changing a setting in an app effects also other apps in same UID. For example disabling internet access of an app also disables the others. It would be nice to list/see/filter apps with UID option.

**Comment by kasnder** [comment]
Yeah, that's right. The Android VPN doesn't differentiate between apps with the same UID. What exactly do you want to do differently? Can you explain some more?

**Comment by Halilb1978** [comment]
Just a listing option to see which apps are in the same UID.

**Comment by AnanasPfirsichSaft** [comment]
NetGuard can do this. And it is really useful. Perhaps it can be done as a search filter like "uid:12345? This wouldn´t clutter the UI too much and of course the UID has to be shown in the app details screen.

I appreciate the simplicity of TC. Don´t add to much... ;)

**Comment by kasnder** [comment]
This is quite an expert feature. Won't implement.
